app/views/pages/about.html.erb

```diff
@@ -75,10 +75,28 @@
       <p><%== t(
         "pages.about.donate-text",
         donate_link: link_to(
-          t("pages.about.donate_link"),
+          t("pages.about.donate-link"),
           "https://www.teaming.net/poreldesarrollodelosbancosdetiempo",
         )
       ).html_safe %></p>
     </div>
   </div>
+  <div class="row">
+    <div class="col-sm-10 col-sm-offset-1 donate">
+      <h2><%= t("pages.about.app-mobile") %></h2>
+      <p><%== t(
```

Comment:
There's no `<%==` in ERB it should be `<%=` instead (check out "Recognized Tags" in http://ruby-doc.org/stdlib-1.9.3/libdoc/erb/rdoc/ERB.html). Can you please, update the other blocks in this file?